llvm/include/llvm/CodeGen/TargetInstrInfo.h

```diff
@@ -682,7 +683,8 @@ class TargetInstrInfo : public MCInstrInfo {
   /// If \p BytesRemoved is non-null, report the change in code size from the
   /// removed instructions.
   virtual unsigned removeBranch(MachineBasicBlock &MBB,
-                                int *BytesRemoved = nullptr) const {
+                                int *BytesRemoved = nullptr,
```

Comment:
You could put `Indexes` _before_ `BytesAdded`/`BytesRemoved`, since only one client (BranchRelaxation) cares about the latter. But I guess it doesn't matter.

Reply:
Done